src/Goerli-DssSpell.t.sol

```diff
@@ -144,28 +135,12 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertTrue(spell.done());
 
         checkIlkClipper(
```

Comment:
this should be use also to test `RENBTC-A` Offboarding

Reply:
It's failing with: `"OSM/contract-not-whitelisted"` after `dog.bark() -> spotter.kick() -> OSM.peek()`
`Spotter` is not whitelisted in the OSM?

Reply:
https://goerli.etherscan.io/tx/0xaeeb62fee6bb75e3162254c8ca57df7af81fadbc4381d9d44e87a73ca96fa956
I think we whitelist a wrong clipper: whitelisted one is for `WBTC`

Reply:
Actually, they use the same oracle (OSM). 
Should I whitelist RENBTC_CLIP in RENBTC_PIP?